catalog-item.html

```diff
@@ -45,15 +60,17 @@ <h1 class="main-heading">ITEM DETAIL</h1>
             <div class="breadcrumbs">
                 <nav class="breadcrumbs-nav">
                     <!-- I understand this separator {&#10072;} can be styled with css instead of symbol -->
-                    <a href="index.html" class="breadcrumb-link">HOME</a> &#10072;
+                    <a href="index.html" class="breadcrumb-link">HOME</a>
+                    <svg width="2" height="16">

```

Comment:
This is another example where it’s better to add the separator as a pseudo-element, especially in breadcrumbs where you may have many items. By using a pseudo-element, you can ensure that the separators between items are added automatically, keeping your HTML cleaner and your code more maintainable.